src/collective/cover/upgrades/v14/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from collective.cover.logger import logger
+from plone import api
+from zope.schema.interfaces import WrongContainedType
+
+
+def register_tile_calendar(setup_tool):
+    """Register Tile Calendar."""
+    registered_tiles = api.portal.get_registry_record(name='plone.app.tiles')
+    registered_tiles.append('collective.cover.calendar')
+    try:
+        api.portal.set_registry_record(
+            name='plone.app.tiles', value=registered_tiles)
+    except WrongContainedType:
```

Comment:
@rodfersou you're probably getting this because the value must be a `set()` and not a `list`.